test/integration/patients/patient/patient-flow.js

```diff
@@ -1014,6 +1014,60 @@ context('patient flow page', function() {
       .find('.js-select')
       .click();
 
+    cy
+      .get('@firstRow')
+      .should('have.class', 'is-selected');
+
+    cy
+      .get('@firstRow')
+      .find('.js-select')
+      .click();
+
+    cy
+      .get('@firstRow')
+      .should('not.have.class', 'is-selected');
+
+    cy
+      .get('@firstRow')
+      .find('.js-select')
+      .click();
+
+    cy
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.relationships.state.data.id = '22222';
+
+        return fx;
+      });
+
+    cy
+      .get('@firstRow')
+      .find('.patient__action-icon')
+      .click();
+
+    cy
+      .get('@firstRow')
+      .find('.js-select')
+      .click();
+
+    cy
+      .get('@firstRow')
+      .should('have.class', 'is-selected');
+
+    cy
+      .get('@firstRow')
+      .find('.js-select')
+      .click();
+
+    cy
+      .get('[data-sidebar-region]')
+      .find('.js-close')
+      .click();
+
+    cy
+      .get('@firstRow')
+      .should('have.class', 'is-selected');
+
```

Comment:
This test verifies that the `.is-selected` class is added/removed when an action is selected/un-selected via its checkbox. 

And that the `.is-selected` class remains on the action in the list if its either selected or being edited via the action sidebar.

**Sidenote**: I wasn't 100% sure where in this file to add these tests. When we do the `click`+`shift` multi-select for the patient flow page, I figured we could create a `specify('click+shift multiselect')` and move these changes into there.